webapp/src/utils/jira_issue_metadata.tsx

```diff
@@ -95,7 +100,7 @@ export function getFields(metadata: IssueMetadata | null, projectKey: string | n
         return {};
     }
 
-    const issueType = getIssueTypes(metadata, projectKey).find((it) => it.id === issueTypeId);
+    const issueType = getIssueTypes(metadata, projectKey, true).find((it) => it.id === issueTypeId);
```

Comment:
Then this becomes

```suggestion
    const issueType = getIssueTypes(metadata, projectKey, {includeSubtasks: true}).find((it) => it.id === issueTypeId);
```

I trust your judgment on if this improves things